scrubber_test.go

```diff
@@ -2,7 +2,11 @@ package metascrubber
 
 import (
 	"bytes"
+	"image"
+	_ "image/jpeg"
+	_ "image/png"
```

Comment:
Looks like these imports aren't necessary.  Maybe the IDE did this for some reason?

Reply:
Oh it's go's weird side effects import thing. In order to `image.Decode` a png or jpeg, you have to additionally import the `image/jpeg` or `image/png` package.

Reply:
Uh wha?  I don't think I've ever run into that and everything compiles / passes on my machine when I remove those imports...  What error do you get when you remove these?

Reply:
Oh, I do know what you're referring to actually.  I guess those packages must have `init` functions that need to run?  The tests are passing now without those side-effect imports though, so maybe they're no longer necessary?

Reply:
Yeah I see that in the doc for the `image` package.  Now I'm wondering why it's working without the imports lol

Reply:
Oh wow not having them in `imports` really seems like it should not work...

Reply:
Right?  Weird haha

Reply:
Well, up to you then!

Reply:
Ohhhhh I know what's happening! So if you look at the `checkImageValidity` function in the test suite: https://github.com/getlantern/meta-scrubber/pull/2/files#diff-a8350e0efa46cd2a8fee043e538b3c2fR73-R82

If the `image.Decode` function fails on the original image, then it will just proceed to the next one and skip attempting to scrub it. That's because there were a couple images in the original corpus I was working with which were just straight up invalid. So if you remove those imports, all `image.Decode` calls will fail and nothing will actually be checked.

I guess that's a bit of a weird situation (like if something in the tests break that causes it to fail the first `image.Decode` then it will just stop being a functional test), but I feel like that's a reasonable tradeoff here...

Reply:
Ohh okay.  Hmm actually, I think that might not be too great in practice.  Someone can add images to the test corpus and not realize that they're just getting skipped.  Just as an example, if I were to implement a GIF scrubber, I would probably just mimic the existing tests and add a bunch of GIFs to the corpus.  The tests would pass and I wouldn't realize that those images were failing to decode and therefore my code wasn't getting tested at all.

Could we fail fast on those broken images?  And just remove bad images from the test corpus?

Reply:
Yeah I think you're probably right about that - I'll throw together a quick PR to address that.